website/docs/getting-started/docker-compose.md

```diff
@@ -0,0 +1,192 @@
+# Docker Compose
+
+The easiest way to run the ORT Server for testing is to use [Docker Compose](https://docs.docker.com/compose/).
+For a proper deployment to Kubernetes, the project will later provide a Helm chart.
+
+:::warning
+Do not use the Docker Compose setup in production as it uses multiple insecure defaults, like providing Keycloak without TLS.
+:::
+
+To use Docker Composer, you need to have [Docker](https://docs.docker.com/get-docker/) and [Docker Compose](https://docs.docker.com/compose/install/) installed.
+Also, you have to clone the ORT Server repository:
+
+```shell
+git clone https://github.com/eclipse-apoapsis/ort-server.git
+```
+
+## Run with Published Images
+
+To start the ORT Server with the required third-party services, run:
+
+```shell
+docker compose up
+```
+
+This will use the ORT Server images [published on GitHub](https://github.com/orgs/eclipse-apoapsis/packages?ecosystem=container).
+By default, the `main` tag is used.
+Those images are built from the `main` branch of the repository.
+To use a different tag, you can set the `ORT_SERVER_IMAGE_TAG` environment variable, for example:
+
+```shell
+ORT_SERVER_IMAGE_TAG=0.1.0-SNAPSHOT-001.sha.aa4d3fa docker compose up
+```
+
+By default, the ORT Server API is exposed on port 8080.
+If this port is already in use, it can be changed using the `ORT_SERVER_CORE_PORT` environment variable:
+
+```shell
+ORT_SERVER_CORE_PORT=8090 docker compose up
+```
+
+When using a different port, please make sure that it is not used by [another service](#accessing-the-services).
+
+### Update the images
+
+Docker Compose will not automatically update the images for a specific tag like `main` once they have been pulled.
+To update the images to the latest version, run:
+
+```shell
+docker compose pull
+```
+
+## Run with Local Images
+
+First, ensure to have [Docker BuildKit](https://docs.docker.com/build/buildkit/) enabled by either using Docker version
+23.0 or newer, running `export DOCKER_BUILDKIT=1`, or configuring `/etc/docker/daemon.json` with:
+
+```json
+{
+  "features": {
+    "buildkit": true
+  }
+}
+```
+
+Then you can build the images with the following command:
+
+```shell
+./gradlew buildAllImages
+```
+
+This will build the images with the `latest` tag.
+Then you can start Docker Compose using the local images by setting the image prefix to an empty string and the tag to `latest`:
+
+```shell
+ORT_SERVER_IMAGE_PREFIX= ORT_SERVER_IMAGE_TAG=latest docker compose up
+```
+
+### Build specific images
```

Comment:
Suggestion: I am not sure whether the information how to build images should be under the Docker Compose topic - at least, I would not have expected to find it there. Should there be a dedicated page explaining how to build the project? The compose documentation could then link to this.